src/contact.ts

```diff
@@ -1,6 +1,12 @@
 export interface Contact {
+  cardType: "person" | "repeater";
   name: string;
+  location: string;
   callsign: string;
-  website?: string;
   star?: boolean;
-}
+  website?: string;
+  frequency?: number;
+  offset?: number;
+  tone?: number;   
+  rxtone?: number;
```

Comment:
1. i'd prefer if the terminology here was consistent -- "rx" and "tx" are ambiguous (rx for who?), i've seen "in tone" and "out tone" though
2. how does this handle CTCSS vs DCS? i don't see a way to disambiguate with this representation. regarding rendering, i've seen DCS codes prefixed with `D` before

Reply:
"like, you usually don't need both name and location, it's usually clear visually what the difference between a CTCSS tone and a DCS code is so you don't need a separate indicator for that" 

Not saying that there shouldn't be a clear distinction, i'm wondering if you have any solutions to this.

Reply:
i think if the user prefixes it with D, then we can assume DCS, otherwise its unstated?

Reply:
in regards to rx/tx i've always seen it in reference to the station tx'ing the repeater or rx from the repeater. (i'm basing this off of what i've seen in chirp)